java/test.sh

```diff
@@ -7,18 +7,19 @@ set -x
 
 ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 
-pushd $ROOT_DIR/../java
-echo "Compiling Java code."
-mvn clean install -Dmaven.test.skip
-
 echo "Checking code format."
 mvn checkstyle:check
 
-echo "Running tests under cluster mode."
-ENABLE_MULTI_LANGUAGE_TESTS=1 mvn test
+echo "Compiling Java code."
+pushd $ROOT_DIR/..
```

Comment:
pushd should come before `mvn checkstyle`, and there should be a pairing popd

Reply:
`$ROOT_DIR/..` is `ray/ folder`, and there is a `pushd ray/java` before `mvn checkstyle`.